docs/source/quasi_dags.ipynb

```diff
@@ -0,0 +1,457 @@
+{
```

Comment:
<p>If we add an example above, it would be nice to see how a RCT actually help us. Again, just to gain some intuition.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/321/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Exactly, and this would would also allow some discussion of the do operator as being an intervention on the causal graph.</p>